src/client/theme-default/styles/vars.css

```diff
@@ -147,8 +147,14 @@
 :root {
   --vp-font-family-base: 'Inter var experimental', 'Inter var', -apple-system,
     BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell,
-    'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
-  --vp-font-family-mono: Menlo, Monaco, Consolas, 'Courier New', monospace;
+    'Fira Sans', 'Droid Sans', 'Helvetica Neue', 'Helvetica', 'Noto Sans',
+    'Source Sans Pro', 'PingFang SC', 'Hiragino Sans GB', 'Noto Sans CJK SC',
+    'Source Han Sans CJK SC', 'PingFang TC', 'Noto Sans CJK TC',
+    'Source Han Sans CJK TC', 'Hiragino Sans', 'Noto Sans CJK JP',
+    'Source Han Sans CJK JA', 'Apple SD Gothic Neo', 'Noto Sans CJK KR',
+    'Source Han Sans CJK KO', sans-serif;
+  --vp-font-family-mono: 'SF Mono', 'Source Code Pro', 'Fira Code', Menlo,
+    Monaco, Consolas, 'Courier New', monospace;
```

Comment:
Personally I would remove Courier New from this list.

If the goal is to show whatever monospace font the OS or browser has configured, using generic names like `--vp-font-family-mono: monospace, monospace;` (need to have a name before the generic `monospace` at the end because of old browser text sizing issues with `font-family: monospace`) is enough.

I read the existing value as an attempt to select a more specific style of monospace font, one with big x-height and no serifs, versus Courier New which has a smallish x-height and slab serifs (and suffers from readability issues at small or medium sizes as a result).

So if none of the listed readable monospace fonts doesn't match, it's probably better to fall back to `monospace` and let the browser decide (which may result in Courier New or something else, e.g. it's definitely not Courier New on Android), rather than explicitly ask for Courier New.